### PR TITLE
Fixed issue #70 - r10k and hiera cron script

### PR DESCRIPTION
**Pull request by pjamenaja**

### CODE REVIEW

**Review by Xtigyro — APPROVED** [review]

**Review by Xtigyro — CHANGES_REQUESTED** [review]
Hey @pjamenaja 👋 ! Thanks for the contribution - we do appreciate it! 🙇 

Could you please update the `CHANGELOG` and also bump the chart version in `Chart.yaml`?


CC: @underscorgan @slconley @Iristyle @scottcressi @mwaggett @nwolfe @adrienthebo @dhollinger @binford2k @alannab

**Comment by pjamenaja** [comment]
Added changes per suggestions above.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version `v0.2.4` is auto-generated.
 
+## [v5.0.3](https://github.com/puppetlabs/puppetserver-helm-chart/tree/v5.0.3) (2020-11-01)
+
+- Fix for r10k_code_cronjob.sh and r10k_hiera_cronjob.sh syntax error with `map[]`
+
+[Full Changelog](https://github.com/puppetlabs/puppetserver-helm-chart/compare/v5.0.1...v5.0.3)
```
Comment (Xtigyro) [inline CHANGELOG.md]: The `v5.0.1` needs to be changed to `v5.0.2`.

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pjamenaja) [reply CHANGELOG.md]: OK. Will fix it and submit again.

**Comment by pjamenaja** [comment]
Fixed per suggestion above.

Thanks.

**Review by Xtigyro — APPROVED** [review]

**Comment by Xtigyro** [comment]
@pjamenaja Would you like your name to be present in the Credits Section in README?

https://github.com/puppetlabs/puppetserver-helm-chart#credits

**Comment by pjamenaja** [comment]
> @pjamenaja Would you like your name to be present in the Credits Section in README?
> 
> https://github.com/puppetlabs/puppetserver-helm-chart#credits

No, thanks.

**Pull request merged**